### DISCUSSION TITLE
Mesh Server error folder Nodes

### DISCUSSION THREAD

**Discussion by bierco2** [Ylianst/MeshCentral#4100] [General]
Good morning,

Today the mesh console does not load, it appears a crypto error, with some alert messages of not loading or configuration in the nodes folder.

![Captura](https://user-images.githubusercontent.com/107195557/172860277-ba27ec4a-eb89-4d1b-9c38-f067f86f8299.PNG)

**Comment by ThiagoCFW** [comment]
Estou com o mesmo problema aqui!
Será que foi alguma atualização? Começou hj!

**Comment by ThiagoCFW** [comment]
https://github.com/Ylianst/MeshCentral/issues/4097#issuecomment-1150594044

**Comment by Ylianst** [comment]
Ha yes, published v1.0.36 with a fix for this. Also, a workaround it to add:

```
"webPageLengthRandomization": false
```

In the settings section of the config.json. You can then update and remove that line. Another solution is to update to NodeJS v16 or higher.

**Reply by bierco2** [reply]
Good morning, now the mesh service does not start, it generates an error at startup. Clarifying that already updated to version 1.0.36

**Reply by bierco2** [reply]
Reinstall and restore and everything OK, thank you very much.
